### ISSUE TITLE
VisIt crashing doing X Ray Image query in parallel

### ISSUE THREAD

**Issue by aowen87** [visit-dav/issues-test#924]
Dan Laney reported that he was running the X Ray Image query in parallel and it was sporadically crashing. He gave me a script and some data files that reproduced the problem on the closed network. This has been broken for quite a while visit 2.4.2 doesn't work properly so it isn't the conversion to double that is causing the problem.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Eric Brugger
Original creation: 07/17/2012 05:20 pm
Original update: 07/20/2012 05:54 pm
Ticket number: 1132
